modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -100,6 +102,34 @@ def conditions_related_to_exposure?
         # This will check 'YES' for box 15A.
         # TODO: for disability mapping ticket
       end
+
+      def treatment_centers
+        @pdf_data[:data][:attributes][:claimInformation] = {}
+        @pdf_data[:data][:attributes][:claimInformation].merge!(
+          treatments: []
+        )
+        treatments = get_treatments
+
+        treatment_details = treatments.map(&:deep_symbolize_keys)
+        @pdf_data[:data][:attributes][:claimInformation][:treatments] = treatment_details
+
+        @pdf_data
+      end
+
+      def get_treatments
+        # Concat treatmentDetails as: Disability Name - Treatment Center Name, City, State
+        @auto_claim['treatments'].map do |tx|
+          center = "#{tx['center']['name']}, #{tx['center']['city']}, #{tx['center']['state']}"
+          details = "#{tx['treatedDisabilityName']}, #{center}"
```

Comment:
slightly different than the request in the ticket. `Disability Name - Treatment Center Name, City, State`, just looks to be a `-` instead of a `,`

Reply:
ok, thanks, I will fix this!